CMakeLists.txt

```diff
@@ -122,25 +122,25 @@ cppyy_make_python_package(PACKAGE pluginplay NAMESPACES pluginplay DEPPACKAGES p
 if("${BUILD_TESTING}")
     set(tests_src_dir "${CMAKE_CURRENT_LIST_DIR}/tests/${PROJECT_NAME}")
     set(examples_src_dir "${CMAKE_CURRENT_LIST_DIR}/tests/${PROJECT_NAME}/docs")
-    cpp_add_library(
+    cmaize_add_library(
         ${PROJECT_NAME}_examples
         SOURCE_DIR ${examples_src_dir}
-        INCLUDE_DIR ${examples_src_dir}
+        INCLUDE_DIRS ${examples_src_dir}
         DEPENDS ${PROJECT_NAME}
     )
     include(nwx_python_mods)
         cppyy_make_python_package(PACKAGE ${PROJECT_NAME}_examples NAMESPACES pluginplay_examples DEPPACKAGES pluginplay)
-    cpp_find_or_build_dependency(
+    cmaize_find_or_build_dependency(
         Catch2
         URL github.com/catchorg/Catch2
         BUILD_TARGET Catch2
         FIND_TARGET Catch2::Catch2
 	    VERSION v2.13.8
```

Comment:
Really minor change, but I think this is tabs instead of spaces, so it ends up getting indented more in some editors. Can you change this? Otherwise, looks great!